e2e/scenario_test.go

```diff
@@ -937,7 +939,7 @@ func Test_ubuntu2204ContainerdURL(t *testing.T) {
 				nbc.ContainerdPackageURL = "https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/m/moby-containerd/moby-containerd_1.6.9+azure-ubuntu22.04u1_amd64.deb"
 			},
 			LiveVMValidators: []*LiveVMValidator{
-				containerdVersionValidator("1.6.9"),
+				mobyComponentVersionValidator("containerd", "1.6.9", "apt"),
```

Comment:
not sure I like this hidden contract `"apt"` shouldnt we pass the OS Family instead ?  from config.VHDUbuntu2204Gen2Containerd.OSSKU ? or something ?